spec/smart_app_launch/token_response_body_test_spec.rb

```diff
@@ -117,15 +117,63 @@ def create_token_request(body: nil, status: 200, headers: nil)
   end
 
   context 'when the fhirContext field is present' do
-    it 'fails if fhirContext is not an Array'
+    it 'passes if fhirContext valid' do
+      numericalElement = 123
+      body = valid_body.merge(fhirContext: ["Organization/123", "DiagnosticReport/123"])
```

Comment:
Could you include a version-specific reference here as well?

http://hl7.org/fhir/R4/references.html#literal